### ISSUE TITLE
Duplicated pod not showing fields

### ISSUE THREAD

**Issue by amars01** [pods-framework/pods#6463]
### Description

Every time I try duplicating a pod, the pod gets duplicated but I can't see any of my fields

If I try re-adding one of my fields I get the following message: “Field not added, it already exists.”

### Version

2.8.10 

### Testing Instructions

1. Pods Admin > Edit pods
2. Duplicate the Pod I want to duplicate
3. A page loads with the duplicated pod, the pod is named like this “podname2”(renamed in the screenshot) but I can’t see my fields. Instead, I get a message saying "There are no fields in this group."

### Screenshots / Screencast

![Screenshot 2022-02-22 201705](https://user-images.githubusercontent.com/38383652/155212066-20c7abb2-5696-4b03-a460-a4dfec3a62b9.png)

### Possible Workaround

Haven't found one

### Site Health Information

```text
`
### wp-core ###

version: 5.9
site_language: en_GB
user_language: en_GB
timezone: Europe/London
permalink: /%postname%/
https_status: true
multisite: false
user_registration: 0
blog_public: 0
default_comment_status: open
environment_type: production
user_count: 4
dotorg_communication: true

### wp-paths-sizes ###

wordpress_path: /var/www/domain
wordpress_size: 52.49 MB (55042085 bytes)
uploads_path: /var/www/domain/wp-content/uploads
uploads_size: 2.19 GB (2348646567 bytes)
themes_path: /var/www/domain/wp-content/themes
themes_size: 8.44 MB (8849097 bytes)
plugins_path: /var/www/domain/wp-content/plugins
plugins_size: 159.72 MB (167477089 bytes)
database_size: 226.91 MB (237928448 bytes)
total_size: 2.62 GB (2817943286 bytes)

### wp-dropins (3) ###

advanced-cache.php: true
db.php: true
object-cache.php: true

### wp-active-theme ###

name: Onepress-Child (onepress-child)
version: 0.1
author: Name
author_website: https://dev.domain
parent_theme: OnePress (onepress)
theme_features: core-block-patterns, widgets-block-editor, automatic-feed-links, title-tag, menus, html5, woocommerce, custom-logo, recommend-plugins, customize-selective-refresh-widgets, wc-product-gallery-zoom, wc-product-gallery-lightbox, wc-product-gallery-slider, editor-styles, align-wide, editor-style, widgets
theme_path: /var/www/domain/wp-content/themes/onepress-child
auto_update: Disabled

### wp-parent-theme ###

name: OnePress (onepress)
version: 2.2.8
author: FameThemes
author_website: http://www.famethemes.com
theme_path: /var/www/domain/wp-content/themes/onepress
auto_update: Disabled

### wp-themes-inactive (1) ###

Twenty Twenty-Two: version: 1.0, author: the WordPress team, Auto-updates disabled

### wp-plugins-active (46) ###

Add to shipping Plan: version: 1.0, author: Name Last name, Auto-updates disabled
Ajax Search Lite: version: 4.9.5, author: Ernest Marcinko, Auto-updates disabled
Amazon Description: version: 1.0, author: Name Last name, Auto-updates disabled
Automations: version: 1.0, author: Name, Auto-updates disabled
Beaver Builder Plugin (Standard Version): version: 2.2.3.3, author: The Beaver Builder Team (latest version: 2.5.2.1), Auto-updates disabled
Content Control: version: 1.1.9, author: Code Atlantic, Auto-updates disabled
Display Returns: version: 1.0, author: Name Last name, Auto-updates disabled
Easy Table of Contents: version: 2.0.17, author: Steven A. Zahm, Auto-updates disabled
Ebay Listing: version: 1.0, author: Name Last name, Auto-updates disabled
EWWW Image Optimizer: version: 6.4.0, author: Exactly WWW (latest version: 6.4.1), Auto-updates disabled
Expenses: version: 1.0, author: Name Last name, Auto-updates disabled
gross_profit_margin: version: 1.0, author: Name Last name, Auto-updates disabled
inventory: version: 1.0, author: Name Last name, Auto-updates disabled
iThemes Security: version: 8.0.2, author: iThemes, Auto-updates disabled
List products on SIte: version: 1.0, author: Name Last name LIVE, Auto-updates disabled
Margin Calc: version: 1.0, author: Name Last name, Auto-updates disabled
Monitor Prices: version: 1.0, author: Name Last name, Auto-updates disabled
Number Reviews Tracker: version: 1.0, author: Name Last name, Auto-updates disabled
Pods - Custom Content Types and Fields: version: 2.8.10, author: Pods Framework Team, Auto-updates disabled
Postie: version: 1.9.59, author: Wayne Allen, Auto-updates disabled
Post Type Switcher: version: 3.2.1, author: Triple J Software, Inc., Auto-updates disabled
Price Audit Log: version: 1.0, author: Name Last name, Auto-updates disabled
Product Research: version: 1.0, author: Name Last name, Auto-updates disabled
Product Tracker: version: 1.0, author: Name Last name, Auto-updates disabled
Profit and Loss: version: 1.0, author: Name Last name, Auto-updates disabled
profit calc: version: 1.0, author: Name Last name, Auto-updates disabled
profit_by_product_type: version: 1.0, author: Name Last name, Auto-updates disabled
Queues: version: 1.0, author: Name Last name, Auto-updates disabled
Ranking Tracker: version: 1.0, author: Name Last name, Auto-updates disabled
Relevanssi: version: 4.14.7, author: Mikko Saari, Auto-updates disabled
Returned Queue: version: 1.0, author: Name Last name, Auto-updates disabled
Revenue By Month: version: 1.0, author: Name Last name, Auto-updates disabled
Reviews Tracker: version: 1.0, author: Name Last name, Auto-updates disabled
Selling Fees calculator camera/smart_watch: version: 1.0, author: Name Last name, Auto-updates disabled
Shipping Costs Phones: version: 1.0, author: Name Last name, Auto-updates disabled
Shipping Plan: version: 1.0, author: Name Last name, Auto-updates disabled
SKU Generation: version: 1.0, author: Name Last name, Auto-updates disabled
Sold Days: version: 1.0, author: Name Last name, Auto-updates disabled
Stock: version: 1.0, author: Name Last name, Auto-updates disabled
Ultimate Member: version: 2.3.0, author: Ultimate Member (latest version: 2.3.1), Auto-updates disabled
Unsold Cogs: version: 1.0, author: Name Last name, Auto-updates disabled
User Role Editor: version: 4.61.1, author: Vladimir Garagulya, Auto-updates disabled
W3 Total Cache: version: 2.2.1, author: BoldGrid, Auto-updates disabled
WP Sheet Editor - Pods: version: 1.0.0, author: WP Sheet Editor, Auto-updates disabled
WP Sheet Editor - Post Types (Premium): version: 2.24.12, author: WP Sheet Editor, Auto-updates disabled
WP User Login Notifier: version: 1.0.5, author: BuddyDev, Auto-updates disabled

### wp-plugins-inactive (5) ###

Docket Cache: version: 21.08.04, author: Nawawi Jamili, Auto-updates disabled
European Market Places: version: 1.0, author: Name Last name, Auto-updates disabled
If Menu - Visibility control for menu items: version: 0.16.2, author: Layered, Auto-updates disabled
WP-Optimize - Clean, Compress, Cache: version: 3.2.2, author: David Anderson, Ruhani Rabin, Team Updraft, Auto-updates disabled
WP Activity Log: version: 4.3.4, author: WP White Security (latest version: 4.3.6), Auto-updates disabled

### wp-media ###

image_editor: EWWWIO_Imagick_Editor
imagick_module_version: 1687
imagemagick_version: ImageMagick 6.9.7-4 Q16 x86_64 20170114 http://www.imagemagick.org
imagick_version: 3.6.0
file_uploads: File uploads is turned off
post_max_size: 300M
upload_max_filesize: 300M
max_effective_size: 300 MB
max_file_uploads: 20
gd_version: 2.2.5
gd_formats: GIF, JPEG, PNG, WebP, BMP, XPM
ghostscript_version: 9.26

### wp-server ###

server_architecture: Linux 4.9.0-16-amd64 x86_64
httpd_software: nginx/1.14.1
php_version: 7.4.27 64bit
php_sapi: fpm-fcgi
max_input_variables: 1000
time_limit: 30
memory_limit: 2048M
max_input_time: 60
upload_max_filesize: 300M
php_post_max_size: 300M
curl_version: 7.52.1 OpenSSL/1.0.2u
suhosin: false
imagick_availability: true
pretty_permalinks: true
htaccess_extra_rules: false

### wp-database ###

extension: mysqli
server_version: 10.1.48-MariaDB-0+deb9u2
client_version: mysqlnd 7.4.27
max_allowed_packet: 16777216
max_connections: 151

### wp-constants ###

WP_HOME: undefined
WP_SITEURL: undefined
WP_CONTENT_DIR: /var/www/domain/wp-content
WP_PLUGIN_DIR: /var/www/domain/wp-content/plugins
WP_MEMORY_LIMIT: 2048M
WP_MAX_MEMORY_LIMIT: 2048M
WP_DEBUG: false
WP_DEBUG_DISPLAY: true
WP_DEBUG_LOG: false
SCRIPT_DEBUG: false
WP_CACHE: true
CONCATENATE_SCRIPTS: undefined
COMPRESS_SCRIPTS: undefined
COMPRESS_CSS: undefined
WP_ENVIRONMENT_TYPE: Undefined
DB_CHARSET: utf8
DB_COLLATE: undefined

### wp-filesystem ###

wordpress: writable
wp-content: not writable
uploads: writable
plugins: writable
themes: writable

### pods ###

pods-server-software: nginx/1.14.1
pods-user-agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/98.0.4758.102 Safari/537.36
pods-session-save-path: /var/lib/php/sessions
pods-session-save-path-exists: Yes
pods-session-save-path-writable: Yes
pods-session-max-lifetime: 1440
pods-opcode-cache-apc: No
pods-opcode-cache-memcached: No
pods-opcode-cache-opcache: Yes
pods-opcode-cache-redis: No
pods-object-cache-apc: No
pods-object-cache-apcu: No
pods-object-cache-memcache: No
pods-object-cache-memcached: Yes
pods-object-cache-redis: No
pods-memory-current-usage: 98.611M
pods-memory-current-usage-real: 21.000M
pods-network-wide: No
pods-install-location: /wp-content/plugins/pods/
pods-developer: No
pods-tableless-mode: No
pods-relationship-table-enabled: Yes
pods-light-mode: No
pods-strict: No
pods-allow-deprecated: Yes
pods-api-cache: Yes
pods-shortcode-allow-evaluate-tags: No
pods-sessions: Disable sessions
pods-can-use-sessions: Yes
pods-relationship-table-status: 1,689

`
```

**Comment by sc0ttkclark** [comment]
Fixed via 60e7784cd554c358291b8a1d509aeed5356fc0aa

**Comment by sc0ttkclark** [comment]
The issue was caused by the code not properly setting the group for the fields being added. The fields still exist on the pod -- and the way to get them to show up is to add `&pods_debug_find_orphan_fields=1` to your edit pod URL and it should automatically fix them for you.
